third_party/move/move-compiler-v2/src/file_format_generator/function_generator.rs

```diff
@@ -985,7 +986,13 @@ impl<'env> FunctionContext<'env> {
 
 impl<'env> BytecodeContext<'env> {
     /// Determine whether the temporary is alive (used) in the reachable code after this point.
-    pub fn is_alive_after(&self, temp: TempIndex) -> bool {
+    /// If `dest_check` is true, temporary should not be written to by the current instruction
```

Comment:
This comment is pretty confusing.  I understand it, but I wonder if we can somehow express it more clearly.  Perhaps:
```
/// If `dest_check` is true then if `temp` is overwritten by the current instruction it is not considered live.
```
In some sense, `dest_check` means that we are interested in whether the value in `temp` from _before_ the instruction is still live after the instruction.  I wonder if that concept could give a more intuitive parameter name, but I can't think of one.

Reply:
I have renamed the parameter and written additional comments.